src/core/errors.ts

```diff
@@ -110,7 +110,7 @@ export class ErrNotEnoughGas extends Err {
 }
 
 /**
- * Signals an invalid value for nince objects.
+ * Signals an invalid value for nonce objects.
```

Comment:
`nonce **objects**` and  `gasLimit **objects**` is not a correct phrasing anymore. Can be simply _gas limit_ and _nonce_.